README.md

```diff
@@ -453,6 +454,9 @@ To get started, simply fork this repo. Please refer to [CONTRIBUTING.md](CONTRIB
 - [Build the Game of Life](https://robertheaton.com/2018/07/20/project-2-game-of-life/)
 - [Create terminal ASCII art](https://robertheaton.com/2018/06/12/programming-projects-for-advanced-beginners-ascii-art/)
 - [Write yourself a Git](https://wyag.thb.lt/)
+- Python API Tutorials-
+  -[Introduction](https://www.dataquest.io/blog/python-api-tutorial/)
+  -[Working with Streaming Data and the Twitter API](https://www.dataquest.io/blog/streaming-data-python/)
```

Comment:
These are not formatted correctly on the rendered markdown:
![image](https://user-images.githubusercontent.com/6811076/65264130-57af0f80-dae4-11e9-82d1-368b81ae8540.png)